homeassistant/components/roborock/coordinator.py

```diff
@@ -40,6 +41,13 @@ def __init__(
         )
         device_info = DeviceData(device, product_info.model, device_networking.ip)
         self.api = RoborockLocalClient(device_info)
+        self.device_specification = DeviceInfo(
```

Comment:
It would be a lot clearer to the reader if we keep the names of the instances in the attributes and variables. Ie, name the `DeviceInfo` instance `self.device_info` and the `DeviceData` instance `device_data` and the `RoborockHassDeviceInfo` instance `roborock_device_info`.

Reply:
Very fair point.

Fixed here: #93843